### ISSUE TITLE
Use Forbidden API and explicit Locale and Charset

### ISSUE THREAD

**Issue by GoogleCodeExporter** [crawler-commons/crawler-commons#66]
```
As explained in [https://code.google.com/p/crawler-commons/issues/detail?id=63] 
we currently rely on the default locale and charset in many places in the code 
which is error prone.

Using [https://code.google.com/p/forbidden-apis/] will help discover such cases 
and fix them.

Patch attached includes the Forbidden API in our pom. Run 'mvn clean test' to 
see the results.  
```

Original issue reported on code.google.com by `digitalpebble` on 23 Jan 2015 at 9:19

Attachments:
* [66-forbidden.patch](https://storage.googleapis.com/google-code-attachments/crawler-commons/issue-66/comment-0/66-forbidden.patch)

**Comment by jnioche** [comment]
There's been a new version of the [forbidden-apis](https://github.com/policeman-tools/forbidden-apis) since. We could maybe run it from the command line [https://github.com/policeman-tools/forbidden-apis/wiki/CliUsage] instead of adding it to the pom.xml

**Comment by lewismc** [comment]
Hi @jnioche this issue has been addressed via #94 ?

**Comment by jnioche** [comment]
yep, thanks
